docs/api/python.rst

```diff
@@ -51,7 +51,7 @@ and rich debugging and accessibility interfaces built-in.
 
 Data Abstractions
 ------------------------------------
-The Folder, Blob, and File APIs provide a simple interface for storing, recalling, and moving data between
+The Folder APIs provide a simple interface for storing, recalling, and moving data between
```

Comment:
maybe we could update this blurb as well? reads a little generic since we were trying to address all the different data abstractions initially but now we only need to describe folder. and we should remove the last sentence here too